### DISCUSSION TITLE
Which programming language should I learn to create a remote monitoring/accountability/spy applicati

### DISCUSSION THREAD

**Discussion by veajawad** [community/community#21375] [Other Feature Feedback, Questions, & Ideas]
<p>Hi</p>
<p>I want to create 2 applications, 1 for computer (Windows) and 2nd for Mobile Device (Android) with the same feature.</p>
<p>I want this application to take a screenshot every hour and save this screenshot in the computer in a password protected folder and that folder is linked with cloud storage.</p>
<p>And user cannot close/stop/delete/uninstall this application without the password.</p>
<p>So which programming language should I learn to create a screenshot monitoring/accountability/spy application for computer &amp; mobile?</p>
<p>I will be really thankful for your answer.</p>
<p>Waiting for your response.</p>

**Comment by h3ct0rjs** [comment, marked as answer]
<p>Hi <a class="mention" href="/veajawad">@veajawad</a> , </p>
<p>I think that you can use python for the application desktop it’s powerful and is okay if you want to learn to code as fast as possible.  You could make a stand alone executable from a Python Script using PyInstaller, if you know about operating systems and os internals you could create a steal mode for you executable. There are so many approaches and methods in computer programming.  </p>
<p>Hope it helps you</p>

**Comment by veajawad** [comment]
<p>Hi!</p>
<p>Thank you so much for your response.</p>
<p>Few people suggest that C# .Net is best for this kind of work.</p>
<p>How do you compare C# &amp; Python? How &amp; why python is better than C#?</p>
<p>Waiting for your response.</p>
<p>Thanks</p>
